src/components/status-image/status-image.css

```
@@ -1,4 +1,21 @@
 .StatusImage {
+    
     border: 1px solid white;
-    background-color: var(--color-busy);
+    height: 423px;
+    width: 414px;
```

Remove border and add max-width 100% to make it more responsive.
We want to position the images using position absolute so we will set the container to position relative
```suggestion
    max-width: 100%;
    width: 414px;
    height: 423px;
    position: relative;
```